conn2res/plotting.py

```diff
@@ -90,34 +88,41 @@ def transform_data(
         # choose features that correspond to largest absolute coefficients
         idx_coef = np.argsort(np.absolute(coef))
         if sum(coef != 0) > n_features:
-            # use top 5 features
+            # use top features
             idx_coef = idx_coef[-1*n_features:]
         else:
-            # use <5 non-zero features
+            # use non-zero features
             idx_coef = np.intersect1d(idx_coef, np.where(coef != 0)[0])
 
         # scale time series with coefficients
         data = data[:, idx_coef]
         if data.size > 0:
             data = data @ np.diag(coef[idx_coef])
-            # data = np.sum(
-            #     data @ np.diag(coef[idx_coef]), axis=1).reshape(-1, 1)
 
     # select given features
     if idx_features is not None:
         data = data[:, idx_features]
 
+    # sum features
+    if feature_set == 'coeff_sum':
+        data = np.sum(data, axis=1).reshape(-1, 1)
+
     # scale features
     if scaler is not None:
-        if scaler == 'l1-norm':
-            scaler = norm(data, ord=1, axis=0)
+        # scalers provided by sklearn.preprocessing
+        if hasattr(preprocessing, scaler):
+            func = getattr(preprocessing, scaler)
+            data = func(data, **kwargs)
```

Comment:
I decided to add this here because I also started using sklearn scaling elsewhere in my pipeline, so  it's good to have it in one place. When you call transform_data only with a scaler argument (and potentially with some kwargs to be passed to sklearn scaler) then it is equivalent to calling the sklearn scaler.

Reply:
Nice!